### ISSUE TITLE
okteto wla error 

### ISSUE THREAD

**Issue by geeks121** [TheCaduceus/WARP-UNLIMITED-ADVANCED#28]
```
2022-11-02 03:54:23.44 UTCwarp-unlimited-8d47dfc77-mfnh7warp-unlimitedHit CTRL-C to stop the server
2022-11-02 03:54:23.44 UTCwarp-unlimited-8d47dfc77-mfnh7warp-unlimited
2022-11-02 03:54:28.34 UTCwarp-unlimited-8d47dfc77-mfnh7warp-unlimitedTraceback (most recent call last):
2022-11-02 03:54:28.34 UTCwarp-unlimited-8d47dfc77-mfnh7warp-unlimited File "/app/warp-plus.py", line 78, i
```

suggestion is to add option to use warp.py using 1 or 0

**Comment by TheCaduceus** [comment]
@geeks121 This issue is not resolved in v4.5.7

**Comment by geeks121** [comment]
Solved,, thanks you
